extensions/wikia/ArticleVideo/ArticleVideoContext.class.php

```diff
@@ -56,4 +57,9 @@ public static function getRelatedVideoData( $title ) {
 	public static function isRelatedVideoEmbedded( $title ) {
 		return !empty( static::getRelatedVideoData( $title ) );
 	}
+
+
+	private static function isArticlePage() {
```

Comment:
even better: https://github.com/Wikia/app/blob/efb59c78e5eb20a32473581dd4643e3273eceb33/includes/wikia/WikiaPageType.class.php#L219-L219
I missed it last time I was looking there